release-notes/SupportedFeatures-v1.0.0-rc1.md

```diff
@@ -28,9 +28,9 @@ Supported features
 ||NetHttpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
 ||NetTcpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
```

Comment:
Good point. Let's put more thoughts in this and not block this PR by it.

BTW, can we update the OS X column from :warning: to :question: